catalog/spatial/registry/registry-common/src/main/java/org/codice/ddf/registry/common/RegistryConstants.java

```diff
@@ -16,6 +16,8 @@
 public class RegistryConstants {
     public static final String REGISTRY_TAG = "registry";
 
+    public static final String REGISTRY_TAG_INTERNAL = "registry.internal";
```

Comment:
Good point.

If we're going to have multiple internal tags, (it seems like we do) it might make sense to invert it though. `internal.registry`, `internal.workspace`, `internal.versioning`, etc.